### PR TITLE
Skip to main content margin

### PR DESCRIPTION
**Pull request by xander-marjoram**
Add new sass file to cover the 'skip to main' element, and set its margin to 4px.

### CODE REVIEW

File: src/scss/partials/_skipToMain.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.skipTo {
```
Comment (ashleynolan) [inline src/scss/partials/_skipToMain.scss]: If you want this to respect a `margin-top` property of `4px`, think you would also need to make the `skipTo` link `display: block` else it'll only get the side margins.

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@justeat/f-header",
   "description": "Fozzie Header – Header Component for Just Eat projects",
-  "version": "0.11.1",
+  "version": "0.12.1",
```
Comment (DamianMullins) [inline package.json]: Should be 0.12.0

Comment (ashleynolan) [inline package.json]: The version would be `0.12.0` – once the minor version increments it sets the patch back to zero

Comment (ashleynolan) [reply package.json]: This now needs to be v0.13.0 as a new release has been published

File: src/scss/partials/_skipTo.scss [hunk 3]
```diff
@@ -0,0 +1,15 @@
+/**
+ * Skip To Styling
+ * =================================
+ * Styles relating to the "skip link" element.
+ */
+
+ .skipTo {
```
Comment (DamianMullins) [inline src/scss/partials/_skipTo.scss]: @ashleynolan This class doesn't use the `c-` prefix, should it? I noticed that the language switcher also doesn't use the prefix.

**Pull request merged**